libs/ui/packages/react/src/components/message/Alert/index.tsx

```diff
@@ -91,28 +103,33 @@ export default function Alert({
   containerProps,
 }: AlertProps): JSX.Element {
   const theme = useTheme();
-  const { color, background } = getColors({ theme, type });
+  const { iconColor, background } = getColors({ theme, type });
   const textProps: { variant?: TextVariants; fontWeight?: string } = {
     variant: "paragraph",
     fontWeight: "medium",
   };
+  const textColor = "neutral.c100";
   return (
     <StyledAlertContainer
```

Comment:
The icon and the other content are supposed to be aligned at the top. You can change StyledAlertContainer and put `align-items: flex-start;`

Reply:
<img width="429" alt="Screenshot 2023-04-25 at 10 40 51" src="https://user-images.githubusercontent.com/91890529/234222687-07b5675d-a972-48e5-b0bb-f9def03241e1.png">